algo/src/main/java/com/hartwig/serve/actionability/ActionableCodonUrlConsolidator.java

```diff
@@ -0,0 +1,23 @@
+package com.hartwig.serve.actionability;
+
+import com.google.common.collect.Sets;
+import com.hartwig.serve.datamodel.range.ActionableCodon;
+import com.hartwig.serve.datamodel.range.ImmutableActionableCodon;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Set;
+
+public class ActionableCodonUrlConsolidator implements UrlConsolidator<ActionableCodon>{
+
+    @NotNull
+    @Override
+    public ActionableCodon stripUrls(@NotNull final ActionableCodon instance) {
+        return ImmutableActionableCodon.builder().from(instance).evidenceUrls(Sets.newHashSet()).build();
```

Comment:
I think your suggestion isn't possible because we receive an infinite exception message. I have try to solve it with a try/catch but looks it didn't work 
![Screenshot 2023-07-21 at 15 35 09](https://github.com/hartwigmedical/serve/assets/32543462/5a039cb5-b3ea-4a6b-9c46-b4d4f430f3d6)

Reply:
There's infinite recursion in your screenshot because `buildWithUrls()` is calling `buildWithUrls()`. I meant that `stripUrls` could call `buildWithUrls()` without changing `buildWithUrls()`. This would just clean up some very minor code duplication, so I don't have any concerns with leaving it the way it is.

Reply:
I mis interpret you. I have update all de UrlConsolidator classes with your suggestion